src/main/java/eu/enmeshed/model/relationships/RelationshipAuditLog.java

```diff
@@ -12,31 +13,20 @@
 @Getter
 @Setter
 @Builder
-public class RelationshipChange {
-
-  private String id;
-
-  private Type type;
-
-  private Status status;
-
-  private RelationshipChangeRequest request;
-
-  private RelationshipChangeResponse response;
-
-  public enum Type {
-    @JsonProperty("Creation")
-    CREATION
-  }
+public class RelationshipAuditLog {
+  private ZonedDateTime createdAt;
+  private String createdBy;
+  private String createdByDevice;
+  private String Reason;
+  private Status oldStatus;
+  private Status newStatus;
 
   public enum Status {
```

Comment:
you should share the RelationshipStatus from Relationship b/c the values are the same. You should then move this to a new file.

Reply:
Done